assets/js/script.js

```diff
@@ -1,5 +1,5 @@
 //  VARIABLES
-var APIKEY = "5275511254394cfab5968fa4a70ea78e"
```

Comment:
maybe we should leave the APIKEY empty for now cause this is causing a conflict now